src/driver/gen_rpc_driver_ssl.erl

```diff
@@ -88,7 +88,7 @@ send(Socket, Data) when is_tuple(Socket), is_binary(Data) ->
 
 -spec activate_socket(ssl:sslsocket()) -> ok | {error, inet:posix()}.
 activate_socket(Socket) when is_tuple(Socket) ->
-    ssl:setopts(Socket, [{active, once}]).
+    ssl:setopts(Socket, [{active, true}]).
```

Comment:
why not N ?

Reply:
For the symmetry: the TCP transport uses `true`.

Reply:
Actually, upon closer investigation the acceptor won't even work with any other setting.